components/clickup/sources/new-list/new-list.mjs

```diff
@@ -0,0 +1,31 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  key: "clickup-new-list",
+  name: "New List (Instant)",
+  description: "Emit new event when a new list is created",
+  version: "0.0.1",
+  dedupe: "unique",
+  type: "source",
+  methods: {
+    ...common.methods,
+    _getMeta({ list_id: listId }) {
+      return {
+        id: listId,
+        summary: listId,
```

Comment:
Again, giving a summary more than `listId` might be better for user experience.

Reply:
The webhook payload doesn't provide more information :(